config/datanode.example.toml

```diff
@@ -25,7 +25,7 @@ heartbeat_timeout_millis = 500
 # Operation timeout in milliseconds, 3000 by default.
 timeout_millis = 3000
 # Connect server timeout in milliseconds, 5000 by default.
-connect_timeout_millis = 5000
+connect_timeout_millis = 1000
```

Comment:
Can we also modify this default value to 1s? https://github.com/GreptimeTeam/greptimedb/blob/621c6f371b0ba952a0432c43baf8258d0fe65e7a/src/common/grpc/src/channel_manager.rs#L33